pkg/cmd/utils.go

```diff
@@ -254,3 +254,77 @@ func CheckPluginImageRepo(kubeClient kubernetes.Interface, ns string, defaultIma
 
 	return resultImage, err
 }
+
+
+// If there is no configmap velero-vsphere-plugin-block-list created before, create a new configmap from the default blocking list.
+// If there is already a configmap velero-vsphere-plugin-block-list created, leave the previous one unchanged.
+func CreateBlockListConfigMap(kubeClient kubernetes.Interface, veleroNs string, resourceToBlock map[string]bool) error {
```

Comment:
What if user deleted the ConfigMap and rebooted the plugin?

Reply:
This logic is called during starting init container. If user deletes the ConfigMap, then reboot the plugin(scale down and up pods), the ConfigMap will be recreated during starting the init containers.

This is only executed once during initialing containers, for every single backup, it will retrieve the existed configmap, if no configmap found, assuming there is no resource to be blocked.